src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -168,6 +169,9 @@ private String getBrowser() {
 
     private FirefoxOptions buildFirefoxOptions(TestName testName) throws IOException {
         FirefoxOptions firefoxOptions = new FirefoxOptions();
+        firefoxOptions.setImplicitWaitTimeout(Duration.ofSeconds(IMPLICIT_WAIT_TIMEOUT));
+        firefoxOptions.setPageLoadTimeout(Duration.ofSeconds(PAGE_LOAD_TIMEOUT));
```

Comment:
probably not needed, but noticed in the exceptions the driver had `0` as the page load timeout. just trying this to rule anything out.